kiwi/schema/kiwi.rnc

```diff
@@ -1248,6 +1248,14 @@ div {
             sch:param [ name = "attr" value = "xen_server" ]
             sch:param [ name = "types" value = "oem vmx" ]
         ]
+    k.type.btrfs_quota_groups.attribute =
+        ## activate the quota system if the filesystem is btrfs based.
+        ## By default the quota system is inactive
+        attribute btrfs_quota_groups { xsd:boolean }
+        >> sch:pattern [ id = "btrfs_quota_groups" is-a = "image_type"
+            sch:param [ name = "attr" value = "btrfs_quota_groups" ]
+            sch:param [ name = "types" value = "oem vmx docker" ]
```

Comment:
I don't see why docker is there... I believe it should only be enabled for disk images, thus oem and vmx only.

Reply:
> found that btrfs_root_is_readonly_snapshot as well as btrfs_root_is_snapshot is also targeted to be valid for docker. I also think that's wrong but wasn't 100% sure

mmm yes that looks like a mistake... Then I guess I am the one to blame :sweat: AFAIR there is no filesystem management at all in containers code.